features/collections/collection_invite.feature

```diff
@@ -1,87 +1,88 @@
-
-Scenario: Inviting a work to an anonymous collection should not make it anonymous
-Given the work "Don't Hide Me"
-  And I have the anonymous collection "anonstuff"
-  And I am logged in as the owner of "anonstuff"
-When I view the work "Don't Hide Me"
-  And I follow "Add To Collections"
-  And I fill in "collection_names" with "anonstuff"
-  And I press "Add"
-Then I should see "This work has been invited to your collection"
-When I view the work "Don't Hide Me"
-Then I should not see "Anonymous"
-
-Scenario: Inviting a work to a hidden collection should not make it unrevealed
-Given the work "Don't Hide Me"
-  And I have the hidden collection "hiddenstuff"
-  And I am logged in as the owner of "hiddenstuff"
-When I view the work "Don't Hide Me"
-  And I follow "Add To Collections"
-  And I fill in "collection_names" with "hiddenstuff"
-  And I press "Add"
-Then I should see "This work has been invited to your collection"
-When I am logged out
-  And I view the work "Don't Hide Me"
-Then I should not see "This work is part of an ongoing challenge and will be revealed soon!"
+@collection @works
+
+Feature: Collection
+  In order to have a collection full of curated works
+  As a collection maintainer
+  I want to add and invite works to my collection
+
+  Scenario: Invite a work to a collection where a user auto-approves inclusion
+  Given I am logged in as "Scott" with password "password"
+    And I set my preferences to automatically agree to my work being collected
+    And I post the work "A Death in Hong Kong"
+  When I have the collection "scotts collection" with name "scotts_collection"
+    And I am logged in as "moderator"
+    And I add the work "A Death in Hong Kong" to the collection "scotts collection"
+  Then I should see "Added to collection(s): scotts collection."
+    And 1 email should be delivered to "Scott"
+    And the email should contain "you have previously elected to allow"
+  When I am logged in as "moderator"
+    And I go to "scotts collection" collection's page
+    And I follow "Manage Items"
+    And I follow "Approved"
+  Then I should see "A Death in Hong Kong"
+
+
+  Scenario: Invite a work to a collection where a users approves inclusion
+  Given I am logged in as "Scott" with password "password"
+    And I set my preferences to require my approval for my work to be collected
+    And I post the work "Murder in Milan" with fandom "Murder She Wrote"
+  When I have the collection "scotts collection" with name "scotts_collection"
+    And I am logged in as "moderator" with password "password"
+    And I add the work "Murder in Milan" to the collection "scotts collection"
+  Then I should see "This work has been invited to your collection (scotts collection)."
+    And 1 email should be delivered to "Scott"
+    And the email should contain "If in future you would prefer to automatically approve requests to add your"
+  When I go to "scotts collection" collection's page
+    And I should see "Works (0)"
```

Comment:
There's an existing issue with the Whens/Thens here, but we should tweak it as long as we're editing this test:
```
Then I should see "Works (0)"
When I follow "Manage Items"
  And I follow "Invited"
```